ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -528,10 +536,80 @@ protected void initTreeTable() {
     doInit();
   }
 
+  private static Comparator<Task> beginComparatorAsc = new Comparator<Task>() {
```

Comment:
I have no access to Task in TaskDefaultColumn bacause it is in other module, so I can't create comparator there